integration-test/helpers/harness.js

```diff
@@ -21,7 +21,7 @@ const setup = async (ops) => {
     const dataDir = fs.mkdtempSync(tmpDirPrefix)
     /** @type {import('puppeteer').PuppeteerLaunchOptions} */
     const puppeteerOps = {
-        headless: 'chrome'
+        headless: true
```

Comment:
@kzar this was your change, but the type seems wrong ([from the docs](https://pptr.dev/api/puppeteer.browserlaunchargumentoptions)). Is this change OK?